examples/example_plot_UEs.py

```diff
@@ -4,7 +4,7 @@
 import elephant.unitary_event_analysis as ue
 import neo
 import matplotlib.pyplot as plt
-from viziphant.unitary_event_analysis import *
+from viziphant.unitary_event_analysis import (load_gdf2Neo, plot_UE, plot_params_default)
```

Comment:
plot_params_default is never used, so it does not need to be imported

Reply:
Thank you.

Since I found a few unrelated things to fix on Viziphant, I will close this pull request and resubmit it as a feature branch with proposed corrections alongside other pull requests.